crates/swc_ecma_parser/Cargo.toml

```diff
@@ -19,7 +19,7 @@ bench = false
 [features]
 # Used for debugging
 debug         = ["tracing-spans"]
-default       = ["typescript", "stacker"]
```

Comment:
Don't you need `swc_common/stacker`? I think the CI may fail

Reply:
Unneeded I think, `swc_ecma_lexer/stacker` is default-enabled.

Reply:
Then `swc_ecma_parser` should depend on `swc_ecma_lexer` with `default-features = false` and this should depend on `swc_ecma_lexer/default` instead.

Reply:
Can you change this, too?